crm_salesperson_planner_re_assign/models/crm_salesperson_planner_visit_reassign.py

```diff
@@ -0,0 +1,208 @@
+# Copyright 2022 Sygel - Manuel Regidor
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class CrmSalespersonPlannerVisitReassign(models.Model):
+    _name = "crm.salesperson.planner.visit.reassign"
+    _description = "Salesperson Planner Visit Reassign"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+
+    name = fields.Char(string="Name", default="/", readonly=True, copy=False)
+    state = fields.Selection(
+        selection=[
+            ("draft", "Draft"),
+            ("validated", "Validated"),
+            ("done", "Done"),
+            ("cancel", "Cancelled"),
+        ],
+        string="Status",
+        readonly=True,
+        copy=False,
+        tracking=True,
+        default="draft",
+    )
+    salesperson_reassign_line_ids = fields.One2many(
```

Comment:
Why using one2many and needing a lot of related/compute fields instead of putting a many2many and show results in list? Having 2 m2m is not a great cost and serves also for splitting templates from the other model.

Reply:
Sorry, but I don't get what you mean.

Reply:
I'm talking about having many2many fields instead of one2many.

Reply:
Done